singularity-eos/eos/eos_davis.hpp

```diff
@@ -193,6 +200,10 @@ class DavisProducts : public EosBase<DavisProducts> {
   DavisProducts(const Real a, const Real b, const Real k, const Real n, const Real vc,
                 const Real pc, const Real Cv)
       : _a(a), _b(b), _k(k), _n(n), _vc(vc), _pc(pc), _Cv(Cv) {}
+  PORTABLE_INLINE_FUNCTION
+  void CheckParams() const {
+    // TODO(JMM): Stub.
```

Comment:
Can you create an issue to just go through these and add more conditions? No need to assign yourself.

Reply:
:+1: #411